packages/material-ui/src/FormControl/FormControl.js

```diff
@@ -28,7 +28,13 @@ export const styles = {
   fullWidth: {
     width: '100%',
   },
-};
+  multiline: {},
+  outlined: {
+    '&$multiline': {
+      backgroundColor: theme.palette.background.paper,
```

Comment:
This bit is wonky. In MCW the label background is hard coded to `#FFF`. I've instead moved that styling to the root and allowed the label to inherit. Ideally the scroll window would be sized appropriately, but I cannot figure out how to do so reliably on this field.

Reply:
I made a mistake- this should have been applied to the input. Unfortunately that means setting white as the default in two places

Reply:
I don't believe the label should have a background color. If you're referring to their Textarea, it's broken:

![image](https://user-images.githubusercontent.com/357702/42532215-15710f14-847e-11e8-8326-70fdce13468c.png)

Reply:
Yes, I absolutely agree. I'm not sure how to solve for this as the input is sized to the outline. This is also causing issues with dynamically resized multiline behaviors. Additionally, their use of a border on the `div` and the `textarea` is causing some artifacting with the inset. Any ideas?